hstream/src/HStream/Server/Handler/Subscription.hs

```diff
@@ -112,22 +112,10 @@ deleteSubscriptionHandler ServerContext {..} (ServerNormalRequest _metadata req@
   modifyMVar_ subscribeRuntimeInfo $ \store -> do
     case HM.lookup deleteSubscriptionRequestSubscriptionId store of
       Just infoMVar -> do
-        modifyMVar infoMVar removeSubscriptionFromZK >>= \case
-          True -> do
-            modifyMVar_ subscriptionCtx (return . Map.delete (T.unpack deleteSubscriptionRequestSubscriptionId))
-            return $ HM.delete deleteSubscriptionRequestSubscriptionId store
-          False -> return store
+        modifyMVar infoMVar removeSubscriptionFromZK >> return store
```

Comment:
need to delete `subscriptionId` from store if `removeSubscriptionFromZK` return true